source/documentation/fuseki2/fuseki-config-endpoint.md

```diff
@@ -1,17 +1,8 @@
 ---
-title: Fuseki Configuration - New Syntax
+title: Fuseki Data Service Configuration Syntax
 ---
 
-Apache Jena 3.13.0 introduces a new syntax for the facilities of a Fuseki service.
-
-The [previous syntax](fuseki-configuration.html) is still valid.
-
-The new syntax enables more configuration options:
-
-* setting the context on a per-endpoint basis
-* having multiple operations at the service access point, switching based on
-operation type
-* a more general structure for adding custom services
+Data service provide a number of operations on a dataset. These can be explicitly named endpoints or operations at he URL of the dataset.  New operations can be configured in; these typically have their own named endpoints.
```

Comment:
"Data service provides" or "Data services provide" I think?

Also s/or operations at he/or operations at the/